helix-core/src/main/java/org/apache/helix/model/CloudConfig.java

```diff
@@ -207,9 +209,13 @@ public Builder(CloudConfig cloudConfig) {
       _cloudInfoProcessorName = cloudConfig.getCloudInfoProcessorName();
     }
 
-    public Builder setClusterName(String v) {
-      _clusterName = v;
-      return this;
+    public Builder(boolean enabled, CloudProvider cloudProvider, String cloudID,
```

Comment:
This is not necessary. The functionality is duplicated with set methods.

Reply:
Fixed. Thanks.